### PR TITLE
Add info about installing forge and how to read crash logs

### PR DESCRIPTION
**Pull request by superpowers04**

### CODE REVIEW

File: _help/installing-forge/index.markdown [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+desc: "Installing forge"
+---
+
+# How to install Forge and mods that require Forge <small>(Java Edition only)</small>
+
+Forge is a modloader commonly used for loading and making mods for Minecraft
+
+
+## Installing Forge in the MC launcher
+You'll need Java to install Forge. If you are using Linux then you'll need to make sure that you have Java 8. [See this article to learn how to install Java](/help/installing-java)
```
Comment (LordRalex) [inline _help/installing-forge/index.markdown:14]: This isn't true as of like 1.13 or something for Forge. in regards to Java 8.

File: _help/installing-forge/index.markdown [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+desc: "Installing forge"
+---
+
+# How to install Forge and mods that require Forge <small>(Java Edition only)</small>
+
+Forge is a modloader commonly used for loading and making mods for Minecraft
+
+
+## Installing Forge in the MC launcher
+You'll need Java to install Forge. If you are using Linux and using 1.12.2 or below then you'll need to make sure that you have Java 8. [See this article to learn how to install Java](/help/installing-java)
```
Comment (LordRalex) [inline _help/installing-forge/index.markdown:14]: Remove the linux part, it's not relevant.

File: _help/installing-forge/index.markdown [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+* Go to the [Minecraft Forge website](https://files.minecraftforge.net/)
+* Choose a Minecraft version from the list on the left.
+* Click "Show All Versions"
+* Pick a Forge version from the list. Most of the time, just using the Forge version at the top of the list is sufficient.
+* In the list, click the circular i icon to the right of "Installer" to get a direct download to the file.
```
Comment (LordRalex) [inline _help/installing-forge/index.markdown:20]: Let's not encourage skipping the ads. Forge actually goes out of their way to make sure adfoc gives safer ads, and it's dickish to tell users to skip it unless there's a specific reason (like curl)

File: _help/installing-forge/index.markdown [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+![forge-allversions](/static/images/help/installing-forge/forge-all-versions.png)
+
+---
+
+<br><small>Some older versions of Minecraft have a Windows installer option for Forge, You can use that if you wish.</small>
+
+* If you get an advert, you probably clicked the wrong thing.
```
Comment (LordRalex) [inline _help/installing-forge/index.markdown:27]: Remove this

File: _help/installing-forge/index.markdown [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+![forge-installed](/static/images/help/installing-forge/forge-installed.png)
+
+---
+
+<!-- (If you really enjoy the things the Forge team does, consider supporting them on Patreon to help bring the Forge site another step closer to being advertisement-free: https://www.patreon.com/LexManos) -->
+
+## [Adding mods](#adding-mods) {#adding-mods}
+* [Head over to your .minecraft folder](/help/finding-minecraft-data-folder/)
+* Create a folder in your .minecraft folder named mods if there isn't one already.
+* Browse through the selection of mods from [Curse Forge](https://curseforge.com/minecraft/mc-mods). Download mod .jars that are compatible with your MC version.
```
Comment (LordRalex) [inline _help/installing-forge/index.markdown:44]: CurseForge, not Curse Forge

**Comment by Southpaw1496** [comment]
Most recent commit (2695ce772eb19ca45231da2fe5615eb2d06ac80c) has an updated document by me with fixes to all the comments you made

**Review by LordRalex — APPROVED** [review]

**Pull request merged**